package dev.pyetro.desafioanotaai.domain.products.exception;

public class ProductNotFoundException extends RuntimeException{
}
